includes/admin_settings_tab.php

```diff
@@ -0,0 +1,333 @@
+							] as $key => $value) { ?>
+								<option value="<?php echo $key ?>" <?php selected($tsml_sharing, $key) ?>>
+									<?php echo $value ?>
+								</option>
+							<?php } ?>
+						</select>
+					</form>
+
+					<?php if ($tsml_sharing == 'restricted') { ?>
+									
+							<h3><strong><?php _e('Authorized Apps', '12-step-meeting-list') ?></strong></h3>
+							<p><?php _e('You may allow access to your meeting data for specific purposes, such as the <a target="_blank" href="https://meetingguide.org/">Meeting Guide App</a>.', '12-step-meeting-list') ?></p>
+
+							<?php if (count($tsml_sharing_keys)) { ?>
+								<table class="tsml_sharing_list">
+									<?php foreach ($tsml_sharing_keys as $key => $name) {
+										$address = admin_url('admin-ajax.php?') . http_build_query([
+											'action' => 'meetings',
+											'key' => $key,
+										]);
+									?>
+										<tr>
+											<td><a href="<?php echo $address ?>" target="_blank"><?php echo $name ?></a></td>
+											<td>
+												<form method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+													<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+													<input type="hidden" name="tsml_remove_sharing_key" value="<?php echo $key ?>">
+													<span class="dashicons dashicons-no-alt"></span>
+												</form>
+											</td>
+										</tr>
+									<?php } ?>
+								</table>
+							<?php } ?>
+							<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+						<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+						<div class="input">
+							<input type="text" name="tsml_add_sharing_key" placeholder="<?php _e('Meeting Guide', '12-step-meeting-list') ?>">
+						</div>
+						<div class="btn">
+							<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+						</div>
+					</form>
+				<?php } else { ?>
+					<h3><strong><?php _e('Public Feed', '12-step-meeting-list') ?></strong></h3>
+					<p><?php _e('The following feed contains your publicly available meeting information.', '12-step-meeting-list') ?></p>
+					<p><?php printf(__('<a class="public_feed" href="%s" target="_blank">Public Data Source</a>', '12-step-meeting-list'), admin_url('admin-ajax.php?action=meetings')) ?></p>
+				<?php } ?>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div id="column2" class="col one-third "  >
+		<div class="postbox">
+		<!-- Put Switch UI here -->
+			<div class="inside">
+				<h1><?php _e('Switch UI', '12-step-meeting-list') ?></h1>
+				<div class="settings-card" >
+					<h3><strong><?php _e('User Interface Display', '12-step-meeting-list') ?></strong></h3>
+					<p><?php _e('Please select the user interface design that is right for your site. Choose between our latest design that we call <b>TSML UI</b> or stay with the old standard <b>Legacy UI</b>.', '12-step-meeting-list') ?></p>							
```

Comment:
Instead of "user interface design", how about "meeting listing style"?

Reply:
Not a fan.  I think most admins would know UI stands for User Interface. I would vote to keep UI...